src/modules/ekf2/EKF/control.cpp

```diff
@@ -428,3 +429,12 @@ void Ekf::controlAuxVelFusion()
 		}
 	}
 }
+
+void Ekf::controlGravityFusion()
+{
+	// fuse gravity observation if our overall acceleration isn't too big
+	const float gravity_scale = _accel_vec_filt.norm() / CONSTANTS_ONE_G;
+	if (gravity_scale >= 0.9f && gravity_scale <= 1.1f) {
```

Comment:
I would prefer to have this a continuous function to reduce possible issues with rectification due to vibration this in and out of the 0.9 -> 1.1 g region

Reply:
Make _control_status.flags.fake_pos == true a requirement for this to run

Reply:
It's probably OK given that the usage threshold test is using a low pass filtered quantity.